[BUG] ubx_gnss sample fails to build

**Describe the bug**
Attempting to build the `ubx_gnss` sample fails due to errors while building `ubxlib`, caused by upstream changes that aren't yet reflected in the `CMakeLists.txt` bridle substitutes.

**To Reproduce**
Steps to reproduce the behavior:
1. `west update` in workspace
2. `west build -p -b <any_board> bridle/samples/ubx_gnss`
3. `CMake Error at...`

**Logs and console output**
```
CMake Error at /ssd/bridle-workspace/zephyr/cmake/modules/extensions.cmake:482 (target_sources):
  Cannot find source file:

    /ssd/bridle-workspace/modules/lib/ubxlib/port/platform/common/mbedtls/u_port_crypto.c
Call Stack (most recent call first):
  /ssd/bridle-workspace/bridle/modules/ubxlib/CMakeLists.txt:267 (zephyr_library_sources)


CMake Error at /ssd/bridle-workspace/zephyr/cmake/modules/extensions.cmake:425 (add_library):
  No SOURCES given to target: ubxlib
Call Stack (most recent call first):
  /ssd/bridle-workspace/bridle/modules/ubxlib/CMakeLists.txt:60 (zephyr_library_named)


CMake Generate step failed.  Build files cannot be regenerated correctly.
FATAL ERROR: command exited with status 1: /nix/store/4vq5ggsg1vmfs09r4sqbidmgvqlxrv14-cmake-3.27.8/bin/cmake -DWEST_PYTHON=/nix/store/wwsr136wn3m7vmjf4i648ygxg496r70v-python3-3.11.6-env/bin/python3.11 -B/ssd/bridle-workspace/bridle/samples/ubx_gnss/build -GNinja -DBOARD=mimxrt1010_evk -S/ssd/bridle-workspace/bridle/samples/ubx_gnss
```

--- a/modules/ubxlib/CMakeLists.txt
+++ b/modules/ubxlib/CMakeLists.txt
@@ -59,6 +59,9 @@ endif() # CONFIG_UBXLIB_GEODESIC
 # The actual library we're building
 zephyr_library_named(ubxlib)
 
+# Extra modules
+list(APPEND ZEPHYR_EXTRA_MODULES ${UBXLIB_BASE}/drivers)
+
 # Include paths that should be exposed to applications ("UBXLIB_INC")
 zephyr_include_directories(
     ${UBXLIB_BASE}
@@ -122,12 +125,6 @@ target_include_directories(ubxlib PRIVATE
     ${ZEPHYR_BASE}/include
 )
 
-# Hacking this in directly for now, ultimately
-# we don't want the mbedtls dependency anyway!
-target_include_directories(ubxlib PRIVATE
-    ${ZEPHYR_BASE}/../modules/crypto/mbedtls/include
-)
-
 # Unconditional sources
 add_src_dir(${UBXLIB_BASE}/common/assert/src)
 add_src_dir(${UBXLIB_BASE}/common/at_client/src)
@@ -162,7 +159,6 @@ list(APPEND UBXLIB_SRC
     ${UBXLIB_BASE}/common/network/src/u_network_private_gnss_stub.c
     ${UBXLIB_BASE}/common/network/src/u_network_private_wifi_stub.c
     ${UBXLIB_BASE}/port/clib/u_port_clib_mktime64.c
-    ${UBXLIB_BASE}/port/platform/common/mbedtls/u_port_crypto.c
     ${UBXLIB_BASE}/port/u_port_heap.c
     ${UBXLIB_BASE}/port/u_port_timezone.c
     ${UBXLIB_BASE}/port/u_port_resource.c
@@ -173,6 +169,8 @@ list(APPEND UBXLIB_SRC
     ${UBXLIB_BASE}/port/platform/zephyr/src/u_port_uart.c
     ${UBXLIB_BASE}/port/platform/zephyr/src/u_port_i2c.c
     ${UBXLIB_BASE}/port/platform/zephyr/src/u_port_spi.c
+    ${UBXLIB_BASE}/port/platform/zephyr/src/u_port_ppp.c
+    ${UBXLIB_BASE}/port/platform/zephyr/src/u_port_board_cfg.c
     ${UBXLIB_BASE}/port/platform/zephyr/src/u_port_private.c
 )
 

